aspnetcore/blazor/tooling.md

```diff
@@ -70,6 +72,57 @@ For more information on trusting the ASP.NET Core HTTPS development certificate,
 
 1. The IDE requests that you add assets to build and debug the project. Select **Yes**.
 
+   **Hosted Blazor WebAssembly launch and task configuration**
```

Comment:
Did you mean to put this whole chunk under the Linux pivot?

Reply:
Yes, DR only wants the VSC guidance in the Linux pivot.

However ... **_good point_** in that readers won't find this without some help, so I've updated the cross-link lines to indicate that the guidance is in the **Linux** OS pivot.

Reply:
> only wants the VSC guidance in the Linux pivot

That is weird...

Reply:
I don't think that I tried to talk him out of it :smile:. IIRC, he wanted it to go by OS to guide newbies with MS/.NET on what tooling to use (one tooling pitched per OS) so that the tooling choice isn't confusing.

I may split this out later tho into its own section and show it for each OS. Let's see if anyone hits VSC problems with the VSC launch/tasks config and they **_don't_** find this somewhat buried guidance in the Linux pivot.